### DISCUSSION TITLE
纯小白求助

### DISCUSSION THREAD

**Discussion by MstZzz** [SAGIRI-kawaii/sagiri-bot#417] [Q&A]
![-64d02f931f18470b.png](https://user-images.githubusercontent.com/125751048/225882238-c19540da-e912-42f4-a42a-be8714213b63.png)

请问如何解决🙏🙏🙏🙏

**Comment by nullqwertyuiop** [comment]
可以尝试 [mirai 论坛中的解决方案](https://mirai.mamoe.net/topic/223/%E6%97%A0%E6%B3%95%E7%99%BB%E5%BD%95%E7%9A%84%E4%B8%B4%E6%97%B6%E5%A4%84%E7%90%86%E6%96%B9%E6%A1%88)。

**Reply by MstZzz** [reply]
感谢感谢！
